### ISSUE TITLE
About feature for COCO test set

### ISSUE THREAD

**Issue by KunpengLi1994** [kuanghuei/SCAN#13]
Hi Kuang-Huei,

Thanks for sharing the code! I'm trying to obtain bottom-up-attention features (36 bbox version) for test data in `testall_ids.txt` of MS COCO. Could you please tell me which [tsv file](https://github.com/peteanderson80/bottom-up-attention) you use for generating `testall_ims.npy`? 

2014 Testing Image Features (40K / 9GB),  2015 Testing Image Features (80K / 17GB) or others ?

I have tried both of them by running the code `util/convert_data.py` but find that ether ` image_id ` in these two tsv files does not match the id in `testall_ids.txt` file. Therefore, I could not generate proper `testall_ims.npy` file. 

Thanks!

**Comment by kuanghuei** [comment]
So the the test set (testall_ids.txt) that Andrej Karpathy produced is actually a split from the original COCO validation set. You don't need COCO 2014/2015 test images.

What Andrej did was:

~800k COCO training set -> Karpathy training split
~50k images from COCO val set -> Karpathy validation split, and we usually take the first 10k for image-text matching
Another ~50k images from COCO val set -> Karpathy test split
Rest of the ~300k images from COCO val set are added for training in most of the recent works.

I know it's super confusing. Just want to make a disclaimer that this is not on me:)

**Comment by LetsGoFir** [comment]
> So the the test set (testall_ids.txt) that Andrej Karpathy produced is actually a split from the original COCO validation set. You don't need COCO 2014/2015 test images.
> 
> What Andrej did was:
> 
> ~800k COCO training set -> Karpathy training split
> ~50k images from COCO val set -> Karpathy validation split, and we usually take the first 10k for image-text matching
> Another ~50k images from COCO val set -> Karpathy test split
> Rest of the ~300k images from COCO val set are added for training in most of the recent works.
> 
> I know it's super confusing. Just want to make a disclaimer that this is not on me:)

Hi, could you tell me how to get the Karpathy split raw images, please? I can only find the feature from
https://cs.stanford.edu/people/karpathy/deepimagesent/.
Thanks for your great job!
